sdk/account/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "solana-account"
+description = "Solana Account type"
+documentation = "https://docs.rs/solana-account"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+bincode = { workspace = true, optional = true }
```

Comment:
Making serde and bincode optional won't remove those dependencies for now as solana-program still pulls them in, but by feature gating them from the start we avoid any breaking changes. Later we'll remove the solana-program dep from this new crate